airbyte-cdk/python/airbyte_cdk/sources/declarative/parsers/model_to_component_factory.py

```diff
@@ -435,7 +435,10 @@ def _get_class_from_fully_qualified_class_name(class_name: str) -> Any:
         split = class_name.split(".")
         module = ".".join(split[:-1])
         class_name = split[-1]
-        return getattr(importlib.import_module(module), class_name)
+        try:
+            return getattr(importlib.import_module(module), class_name)
+        except AttributeError:
+            raise ValueError(f"Class {class_name} could not be found.")
```

Comment:
Non-blocking, nitpick on wording.

```suggestion
            raise ValueError(f"Could not load class {class_name}.")
```